test/demo/utils/notificationsRemote.ts

```diff
@@ -5,8 +5,14 @@ import {Events} from '../../../src/client/internal';
 
 import {OFPuppeteerBrowser, BaseWindowContext} from './ofPuppeteer';
 
+type ReceivedEvents = {
+    listenerId: number;
```

Comment:
done

Reply:
Is it intentional that you're using `eventType` when the payload will always have a `type`?

Reply:
I was trying to name it in accordance with the symbol 'eventTypes', as we go 
`for (const eventType of eventTypes) {...` both in fdc3 and here.

Reply:
My comment isn't on the naming, it's on having `eventType` here at all. Is there an argument for not just using `payload.type` rather than storing `eventType` separately? You could then just get rid of `ReceivedEvents` and just use `Events`